app/-private/equal.js

```diff
@@ -0,0 +1 @@
+export { default } from 'ember-redux/-private/equal';
```

Comment:
not excited about it but I did need the ability to unit test it (and couldn't require this w/out it being in app with an export)